report_thesis/src/sections/background.tex

```diff
@@ -5,7 +5,18 @@ \subsubsection{Standard Scaler}
 
 \subsubsection{Max Absolute Scaler}
 
-\subsubsection{MinMax Scaler}
+\subsubsection{Min-Max Normalization}
+Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
+The goal is to normalize the range of the data to a specific scale, typically 0 to 1. 
+Mathematically, min-max normalization is defined as:
+
+$$ 
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
+$$
+
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively, and $new min$ and $new max$ define the range of the transformed data. 
+This type of normalization is particularly useful when algorithms, such as neural networks, are sensitive to the magnitude of input features. 
+Normalizing ensures that each feature contributes equally to the result, leading to more effective and faster convergence.
```

Comment:
to which result?

section reads mostly geared towards neural nets / other algs that use gradient descent, no?